consumers/hidden-test-consumer/src/hidden_test_consumer/process.py

```
@@ -0,0 +1,298 @@
+            self.bundle_and_upload,
+            self.cleanup,
+        ]
+        yield ProcessRequest(
+            status=Status.INFO, message="⚙️ Started processing hidden test data"
+        )
+
+        for step in steps:
+            try:
+                yield from step()
+                sleep(2)
+            except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)